chaudloader/src/hooks/stage0.rs

```diff
@@ -115,6 +118,10 @@ fn init(
 
     let mut loaded_mods = std::collections::HashMap::<String, mods::State>::new();
 
+    assert!(MODAUDIOFILES
+        .set(std::sync::Mutex::new(ModAudioFiles::new()))
+        .is_ok());
```

Comment:
Would this be purely for aesthetic reasons?  This is modifying `MODAUDIOFILES` during the lua script so to move this would take a bit of refactoring.

Reply:
Yeah, I figured it could be moved easily so it would be easier to read if it was structured like the `MODFUNCTIONS` part. If that's not the case and this is simpler then this is perfectly fine.